lib/reek/smells/attribute.rb

```diff
@@ -32,21 +28,19 @@ def self.contexts # :nodoc:
         [:class, :module]
       end
 
-      def self.default_config
-        super.merge(SmellConfiguration::ENABLED_KEY => false)
-      end
-
       #
       # Checks whether the given class declares any attributes.
       #
       # @return [Array<SmellWarning>]
       #
       def examine_context(ctx)
+        @visiblity_tracker = {}
```

Comment:
1. This voolates our new styleguide regarding instance variables..:)
2. Why not leave it in the constructor. Looks to me like it belongs there.